src/sql/workbench/browser/designer/designerActions.ts

```diff
@@ -0,0 +1,56 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { Designer } from 'sql/workbench/browser/designer/designer';
+import { DesignerEdit, DesignerEditType } from 'sql/workbench/browser/designer/interfaces';
+import { Action } from 'vs/base/common/actions';
+import { localize } from 'vs/nls';
+
+export class AddBeforeSelectedColumn extends Action {
+	public static ID = 'tableDesigner.addBeforeSelectedColumn';
```

Comment:
similarly, these actions are hardcoded for database table column, not generic enough.